src/cls/IPM/Main.cls

```diff
@@ -2865,6 +2878,62 @@ ClassMethod EnableIPM(ByRef pCommandInfo)
 	}
 }
 
+ClassMethod CanUpdate(modName As %String, semver As %IPM.General.SemanticVersion) As %Boolean
```

Comment:
@isc-shuliu  Doesn't look like this method is used?

Reply:
Yes, you're right. I haven't gone to that point yet. My plan is to show if an outdated package can be updated to a particular version without violating dependency constraint. This PR is still a Work-In-Progress